3ds Max/Max2Babylon/Exporter/BabylonExporter.Skeleton.cs

```diff
@@ -135,10 +110,88 @@ private List<IIGameNode> GetRevelantNodes(IIGameSkin skin)
                 }
             }
 
-            revelantNodes = sorted;
-            revelantNodesBySkin.Add(skin, revelantNodes);   // Stock the result for optimization
+            revelantNodesBySkin.Add(skin, sorted);   // Stock the result for optimization
+
+            return sorted;
+        }
+
+        private IIGameNode GetCommonAncestor(List<IIGameNode> bones)
+        {
+            IIGameNode ancestor = null;
+            foreach (IIGameNode b in bones)
+            {
+                ancestor = GetCommonAncestor(ancestor, b);
+                if (ancestor == null)
+                {
+                    break;
+                }
+            }
+
+            return ancestor;
+        }
+
+        private IIGameNode GetCommonAncestor(IIGameNode nodeA, IIGameNode nodeB)
+        {
+            if (nodeA == nodeB || nodeB == null) return nodeA;
+            if (nodeA == null) return nodeB;
+
+            List<IIGameNode> previousANodes = new List<IIGameNode>();
+            previousANodes.Add(nodeA);
+            List<IIGameNode> previousBNodes = new List<IIGameNode>();
+            previousBNodes.Add(nodeB);
+
+            int previousANodesIndex = 0;
+            int previousBNodesIndex = 0;
+
+            while (true)
```

Comment:
so here, we populate "previousANodes" and "previousBNodes" with the hierarchy of A and B. Can we change the name to be a bit more descriptive?